queue/queue_private.js

```diff
@@ -39,8 +54,30 @@ queueApp.controller("QueuePrivateController", function($scope, $controller, Queu
 	$scope.cancelledAppointment = function() {
 		$scope.client.ended = true;
 		QueueService.cancelledAppointment($scope.client.appointmentId);
+	};
+
+	$scope.getVolunteers = function() {
+		let year = $scope.currentDate.getFullYear(),
+		month = $scope.currentDate.getMonth() + 1,
+		day = $scope.currentDate.getDate(),
+		siteId = 1; // TODO THIS NEEDS TO BE CHANGED TO THE SITE SELECTED ONCE THAT IS IMPLEMENTED
```

Comment:
Once @andregarivay gets done with the ticket to make the queue pull for a certain location, this will need to be updated for that.